### DISCUSSION TITLE
Unable to create backup

### DISCUSSION THREAD

**Discussion by jameejay** [pymedusa/Medusa#10817] [Q&A] [answered]
Lately, I've been unable to create a backup and update Medusa. I'm 100 commits behind.

Running on TrueNAS server inside a Jail. 

I can watch the backup file grow in size, using FileZilla, on the server as I refresh the directory.  The backup file size has reached 4,405,660,220. Is that too large for only 46 shows?

Edit: I did look at the logs but didn't see any WARNINGS related to this.

![image](https://user-images.githubusercontent.com/30181160/178855309-47917df7-abcc-42ac-ab57-79584b73b303.png)
![image](https://user-images.githubusercontent.com/30181160/178855637-e25f3534-21d7-41df-80a4-a56458617f5c.png)

**Comment by p0psicles** [comment, marked as answer]
Probaly your cache has grown large.
Might want to also check your *.dbm files. Those can also have grown large. You can delete those, and restart medusa.
Last there is also an option to not include the cache files within the backup. In `config - Advanced settings - backup`

**Reply by jameejay** [reply]
Thanks. And which cache file? Assuming I can delete it? I see the recommended file is big. Perhaps the culprit. 

![image](https://user-images.githubusercontent.com/30181160/179097761-d077e121-c2cd-4899-ae50-575164596caf.png)

**Reply by jameejay** [reply]
Also, no option for advanced to not include cache.

![image](https://user-images.githubusercontent.com/30181160/179097995-e9cf3beb-c93e-4a4c-8e9b-925240a1d057.png)
![image](https://user-images.githubusercontent.com/30181160/179098018-47f622f7-c6eb-49af-b9cd-e339ad65a575.png)

**Reply by medariox** [reply]
Yes, delete the recommended file.

**Reply by p0psicles** [reply]
From next release the recommended.dbm file will not grow as fast.
I've fixed the bug that caused this.

**Comment by jameejay** [comment]
I just noticed I have two Cache directories. The second is super old. 

![image](https://user-images.githubusercontent.com/30181160/179115713-3b46522d-a2c3-4c6e-bc90-70eec9457dab.png)

**Comment by jameejay** [comment]
Thanks all. Worked like a charm!

**Comment by jameejay** [comment]
Drrr...found the advanced tab. Right in front of my eyes. 🤦‍♂️ 

![image](https://user-images.githubusercontent.com/30181160/179116507-a7909aee-a5c2-45a5-82d2-f182fdcb91d6.png)
